src/core/border_router/routing_manager.hpp

```diff
@@ -602,6 +602,39 @@ class RoutingManager : public InstanceLocator
 
 #endif // OPENTHREAD_CONFIG_BORDER_ROUTING_DHCP6_PD_ENABLE
 
+#if OPENTHREAD_CONFIG_BORDER_ROUTING_REACHABLITY_CHECK_ICMP6_ERROR_ENABLE
+    /**
+     * Determines whether to send an ICMPv6 Destination Unreachable error to the sender based on reachability and
+     * source address.
+     *
+     * If the Border Router (BR) decides to forward an IPv6 message to a destination outside the Adjacent Infrastructure
+     * Link (AIL),  and the message's source address matches a BR-generated ULA OMR prefix (with low preference) an
```

Comment:
This paragraph should also mention the condition "and the destination is unreachable" - currently not mentioned - that is also checked before the ICMPv6 message is sent.

Reply:
Updated this comment.